src/styles/scss/style.scss

```diff
@@ -144,41 +168,48 @@ body {
 	font-size: 25px;
 	background: #fff;
 	cursor: pointer;
+
 	#sidebar-close {
 		cursor: pointer;
 		color: $color_2;
 	}
 }
+
 .main {
 	left: 260px;
+	top: 55.98px;
```

Comment:
Did a top offset (based on the size of  header nav bar) to make the main content area not covered by the nav bar.
Kindly check if this is okay Thank you